tensorbay/client/segment.py

```diff
@@ -221,6 +222,25 @@ def _upload_label(self, data: Data, sensor_name: Optional[str] = None) -> None:
             post_data["sensorName"] = sensor_name
         self._client.open_api_do("PUT", "labels", self.dataset_id, json=post_data)
 
+    def delete_data(self, remote_paths: Iterator[str]) -> None:
```

Comment:
`Iterator` should be `Iterable`
Also the input type should be `Union[str, Iterable[str]]` to support deleting one data